app/views/course/assessment/question/scribing/_scribing.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.autogradable question.auto_gradable?
```

Comment:
Just noticed this, missing newline :scream:

Reply:
OK, will be careful next time. we may fix this in next PR. 😨